src/main/java/uk/gov/crowncommercial/dsd/api/auth/model/account/Address.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.crowncommercial.dsd.api.auth.model.account;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+/**
+ * CCS Address.
+ *
+ */
+@Data
+public class Address {
+
+  @JsonProperty("id")
```

Comment:
We could drop most of these explicit `@JsonProperty` annotations I think.. I was in two minds but got loads of them too.